cmd/admin-handlers.go

```diff
@@ -1951,3 +1954,102 @@ func checkConnection(endpointStr string, timeout time.Duration) error {
 	defer xhttp.DrainBody(resp.Body)
 	return nil
 }
+
+// getRawFiler provides an interface for getting raw FS files.
+type getRawFiler interface {
+	GetRawFile(ctx context.Context, volume, file string, fn func(r io.Reader, host string, disk string, filename string, size int64, modtime time.Time) error) error
+}
+
+// DownloadRawHandler - GET /minio/admin/v3/download-raw
+// ----------
+// Download file from all nodes in a zip format
+func (a adminAPIHandlers) DownloadRawHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I was thinking we can name this as `InspectObject` handler @klauspost

Reply:
It is not on object level, it is on file level. Maybe `InspectFile`? (though I do plan to allow wildcards for multiple files at some point) @harshavardhana

Reply:
@klauspost I was thinking from the point of view that we can use this for `mc admin heal inspect` would actually download the content - you can also do `list` a prefix download all of them as well.

Reply:
@harshavardhana I was thinking of making the wildcard support on the server so only a single file would be sent.

I will use the "Inspect" name, though.